MapboxNavigation/ManeuverView.swift

```diff
@@ -83,7 +83,7 @@ open class ManeuverView: UIView {
             ManeuversStyleKit.drawFork(frame: bounds, resizing: resizing, primaryColor: primaryColor, secondaryColor: secondaryColor)
             flip = [.left, .slightLeft, .sharpLeft].contains(direction)
         case .takeRoundabout, .turnAtRoundabout, .takeRotary:
-            ManeuversStyleKit.drawRoundabout(frame: bounds, resizing: resizing, primaryColor: primaryColor, secondaryColor: secondaryColor, roundabout_angle: CGFloat(visualInstruction.primaryInstruction.degrees))
+            ManeuversStyleKit.drawRoundabout(frame: bounds, resizing: resizing, primaryColor: primaryColor, secondaryColor: secondaryColor, roundabout_angle: CGFloat(visualInstruction.primaryInstruction.finalHeading))
```

Comment:
I think this change should be reverted, degrees is specific to roundabouts.

Reply:
This is a temporary change, as the current master copy of MapboxDirections.swift is missing `degrees`. Had to use `finalHeading` till we publish the changes which has the `degrees` 
 field. 
https://github.com/mapbox/MapboxDirections.swift/blob/master/MapboxDirections/MBVisualInstruction.swift#L38

Reply:
Oh wow, indeed it was renamed https://github.com/mapbox/MapboxDirections.swift/commit/03979d6ae59d828ed623217512e9903f8af034cd